### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by ewelin34** [toreamun/victorsmartkill-homeassistant#25] [General]
Had mice get into our pantry about a month ago, after resolving the issues I found your integration for HA and knew I need to get these traps for the next time they get in. Thankfully it only happens about once a year, but now I'm thinking of other places I can put these and it'll be great to have HA notify me if needed. 

Thanks for the great work!

**Comment by toreamun** [comment]
I am glad you find the integrations useful. I find these traps to be very useful in a bit unavailable places like my attic.
